### PR TITLE
Tigers group Svante Ljunggren and Isabell Wästfelt

### PR DESCRIPTION
**Pull request by MattLjunggren**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="stylesheet" href="style.css" />
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Acme&family=Bebas+Neue&family=Chewy&family=Lexend:wght@200&family=Rationale&display=swap" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet" />
+    <title>Bot Theatre</title>
+  </head>
+
+  <body>
+    <h1>Bot Theatre</h1>
```
Comment (JensBergqvist) [inline index.html:16]: Interesting take on a bot!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+          <p>${message}</p>
+        </div>
+      </section>
+    `
+  }
+  chat.scrollTop = chat.scrollHeight
+}
+
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
```
Comment (JensBergqvist) [inline script.js:45]: For the final update, it's better to delete the console, since it's only for testing purpose.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+  } else if (type === 'romance') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Select a romantic movie...</option>
+        <option value="Pride & Prejudice ">Pride & Prejudice</option>
+        <option value="Titanic">Titanic</option>
+        <option value="Notebook">The Notebook</option>
+      </select>
+    `
+  } else if (type === 'horror') {
```
Comment (JensBergqvist) [inline script.js:126]: Very nice structure to the code, it's very easy to follow the steps on what's happening.

File: script.js [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+      <button id="ready">Ready</button>
+    `
+    document
+      .getElementById('ready')
+      .addEventListener('click', () => thankYou('Ready'))
+
+    }
+
+
+const thankYou = () => {
```
Comment (JensBergqvist) [inline script.js:209]: A very nice way to end the conversation with the bot.

File: script.js [hunk 5]
```diff
@@ -0,0 +1,228 @@
...
+  botReply(
+    `So you want to watch some ${type}? Great choice. Select something from the list!`
+  )
+
+  if (type === 'comedy') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>👇 Select a comedy...</option>
+        <option value="My Big Fat Greek Wedding">Big fat greek wedding</option>
+        <option value="Dumb Dumber">Dumb Dumber</option>
```
Comment (JensBergqvist) [inline script.js:113]: Great movie!

File: style.css [hunk 6]
```diff
@@ -0,0 +1,165 @@
+* {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+  }
+  
+  body {
+	font-family: 'Acme';
+	background-image: url(image/cinema.jpg);
```
Comment (JensBergqvist) [inline style.css:9]: Beautiful background!

File: style.css [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+  
+  .user-msg {
+	display: flex;
+	justify-content: flex-end;
+	margin: 16px 0 0 8px;
+	flex-shrink: 0;
+  }
+  
+  .bot-msg img,
+  .user-msg img {
```
Comment (JensBergqvist) [inline style.css:116]: Love the images for the bot and user.

**Review by JensBergqvist — APPROVED** [review]
It's a great looking bot!  The structure is very easy to read and we think you did an awesome job with this!

File: index.html (hunk 1 above)
Comment (eyahya-khan) [reply index.html:16]: Nice work

File: script.js [hunk 8]
```diff
@@ -0,0 +1,227 @@
...
+    `Nice to meet you ${msg}. What type of mood are you in?`
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="happyBtn">🤣</button>
+    <button id="romanticBtn">🥰</button>
+    <button id="horrorBtn">😱</button>
+    <button id="bollywoodBtn">💃🏼</button>
+    `
+
```
Comment (eyahya-khan) [inline script.js:86]: nice to see that you use image, fantastic

File: style.css [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+	padding: 13px 30px;
+	margin-right: 4px;
+	font-size: 20px;
+	line-height: 26px;
+	font-family: 'Acme';
+	font-weight: 500;
+	cursor: pointer;
+	transition: all 0.3s ease;
+  }
+
```
Comment (eyahya-khan) [inline style.css:161]: Clean code.

File: script.js [hunk 10]
```diff
@@ -0,0 +1,227 @@
...
+    questionNumber++
+
+    botReply (`Snacks is life! Whats your favorite ${snack}`)
+
+    inputWrapper.innerHTML = `
+      <button id="popcornBtn">🍿</button>
+      <button id="candyBtn">🍫</button>
+      <button id="cheeseBtn">🧀 </button>
+      `
+    document
```
Comment (eyahya-khan) [inline script.js:182]: easy to understand

**Pull request closed without merging**